src/v2/Apps/_Preferences2/__tests__/PreferencesApp.jest.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen, waitFor } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { flushPromiseQueue } from "v2/DevTools"
+import { PreferencesApp } from "../PreferencesApp"
+
+describe("PreferencesApp", () => {
+  it("renders the preference center", () => {
+    render(<PreferencesApp></PreferencesApp>)
+
+    expect(screen.getByText("Preferences Center")).toBeInTheDocument()
+  })
+
+  it.skip("allows user to uncheck all boxes with unsubscribe all", async () => {
```

Comment:
Any reason we're skipping the test? Also, if there's any other behavior we can test here that would be good. (Totally fine with merging this and returning to tests later tho; don't want to hold up momentum)

Reply:
I looked at this with @The-Beez-Kneez and it seemed like the component wasn't updating (the props looked fine). Not entirely sure what the underlying issue is.

Reply:
Don't want to rehash this right now, but just use enzyme, we get better holistically better test coverage and its easier to write and read and more.

I'm very close to the opinion that we should let devs choose which testing system they want to use. Some might be better for react testing library, but we shouldn't be fighting with tests. We haven't had to in the past, aside from minor stuff like `flushPromiseQueue` hacks, which is entirely minor.

Reply:
I am skipping this for now, because @jacobherrington and I have plans to look at it and write something more covering as we progress. But I did not want to remove the test entirely, so that we have a launch base when we return to it.

If y'all are interested in telling @jacobherrington and I the best path forward for testing react components in Force, we are open to it. Out plan was to add in some Cypress tests.

Reply:
We can add Cypress tests, but we should def be writing unit tests as well for this kind of functionality. If you can cover all behavior using react testing library, feel free, but also feel free to use what works best for you (as long as there's coverage).

Reply:
I will revisit this test and figure out what's going on.

The intention was to cover the functionality described in the cards with unit tests, then add an integration test at the end to catch regressions, etc.

Reply:
For posterity: https://github.com/artsy/force/pull/9291